src/JJMasterData.Core/DataManager/FormFileInfo.cs

```diff
@@ -2,26 +2,11 @@
 
 public class FormFileInfo
 {
-    private FormFileContent _content;
-
-    public FormFileContent Content 
-    {
-        get
-        {
-            if (_content == null)
-                _content = new FormFileContent();
-
-            return _content;
-        }
-        set
-        {
-            _content = value;
-        }
-    }
+    public FormFileContent Content {get; set;} = new();
```

Comment:
If you use Rider decompiler, you can check this is the same thing as a backing field.

Reply:
@gumbarros It's very cool to be able to compare the generated code! 
But if you notice in detail, after the change it instantiates the class in the constructor, and the other one only instantiates the class when the property is called.